otherlibs/systhreads/st_stubs.c

```diff
@@ -15,6 +15,23 @@
 
 #define CAML_INTERNALS
 
+#define _GNU_SOURCE /* helps to find pthread_setname_np() */
+#include "caml/config.h"
+
+#if defined(_WIN32)
+#  include <windows.h>
+#  include <processthreadsapi.h>
+#  include <caml/osdeps.h>
+#elif defined(HAS_PRCTL)
+#  include <sys/prctl.h>
```

Comment:
The manual reads:
```
#include <linux/prctl.h>  /* Definition of PR_* constants */
```
is this header not needed?

Reply:
I think the fact that it compiles is a good proof that it isn't